docs/concepts/safety.md

```diff
@@ -68,9 +68,9 @@ repair replica's local disks.
 - All data in TigerBeetle is immutable, checksummed, and hash-chained, providing a strong guarantee
   that no corruption or tampering happened. In case of a latent sector error, the error is detected
   and repaired without any operator involvement.
-- Most consensus algorithm lose data or become unavailable if the write ahead log gets corrupted.
-  TigerBeetle uses [Protocol Aware Recovery](https://www.youtube.com/watch?v=fDY6Wi0GcPs) to remain
-  available unless the data gets corrupted on every single replica.
+- Most consensus implementations lose data or become unavailable if the write ahead log gets
+  corrupted. TigerBeetle uses [Protocol Aware Recovery](https://www.youtube.com/watch?v=fDY6Wi0GcPs)
+  to remain available unless the data gets corrupted on every single replica.
```

Comment:
This might fall under "online verification" which is mentioned later, but we could also mention the disk scrubber, which proactively looks for LSE's to ensure that we find faults even in cold data.